data-preprocessing/JSON  Section/README.md

```diff
@@ -0,0 +1,12 @@
+# How to use
+
+## Useful Information
+The json files are of two forms. The structure of the Algebra.json is slightly different from Chemistry 2e.json and College Physics 2e.json. As such slightly different block of code handle the return of the sections to account for these differences.
```

Comment:
I find this explanation a bit vague. Would be good to narrow down to exactly what you are talking about and provide a sound and meaningful exegesis of why you needed to implement two blocks of code to handle the related differences